src/twisted/python/htmlizer.py

```diff
@@ -6,23 +6,33 @@
 HTML rendering of Python source.
 """
 
-import tokenize, cgi, keyword
+from twisted.python.compat import _PY3
+
+import tokenize, keyword
+if _PY3:
```

Comment:
`_PY3` conditionals should be avoided as much as possible in application code.  The first line of defense is checking `twisted.python.compat` for existing stuff, and in this case `twisted.python.compat.escape` is already `html.escape` on py3 and `cgi.escape` on py2.